lib/pc/index.js

```diff
@@ -5,13 +5,15 @@ nbt.float = value => ({ type: 'float', value })
 const { networkBiomesToMcDataSchema, mcDataSchemaToNetworkBiomes } = require('./transforms')
 
 module.exports = (data, staticData) => {
+  const id = str => data.supportFeature('registryKeysStartWithMinecraft') ? str : ('minecraft:' + str)
```

Comment:
this feature is not defined in minecraft data

Reply:
Can someone define it for me

Reply:
Make a PR to minecraft-data to add it

But I don't really get what you want to do here since Minecraft prefixes are there for all versions